packages/@aws-cdk/aws-apigateway/test/integ.api-definition.inline.private.ts

```diff
@@ -0,0 +1,67 @@
+import * as cdk from '@aws-cdk/core';
+import * as apigateway from '../lib';
+
+/*
+ * Stack verification steps:
+ * * `curl -i <CFN output PetsURL>` should return HTTP code 200
+ */
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integtest-restapi-fromdefinition-inline-private');
+
+const api = new apigateway.SpecRestApi(stack, 'my-private-api', {
+  apiDefinition: apigateway.ApiDefinition.fromInline({
+    openapi: '3.0.2',
+    info: {
+      version: '1.0.0',
+      title: 'Test API for CDK',
+    },
+    paths: {
+      '/pets': {
+        get: {
+          'summary': 'Test Method',
+          'operationId': 'testMethod',
+          'responses': {
+            200: {
+              description: 'A paged array of pets',
+              content: {
+                'application/json': {
+                  schema: {
+                    $ref: '#/components/schemas/Empty',
+                  },
+                },
+              },
+            },
+          },
+          'x-amazon-apigateway-integration': {
+            responses: {
+              default: {
+                statusCode: '200',
+              },
+            },
+            requestTemplates: {
+              'application/json': '{"statusCode": 200}',
+            },
+            passthroughBehavior: 'when_no_match',
+            type: 'mock',
+          },
+        },
+      },
+    },
+    components: {
+      schemas: {
+        Empty: {
+          title: 'Empty Schema',
+          type: 'object',
+        },
+      },
+    },
+  }),
+  endpointTypes: [apigateway.EndpointType.PRIVATE],
```

Comment:
From the readme, private endpoints need a vpc configured. I don't see that here.

Please ensure this is a deployable cdk app.

Reply:
It doesn't need the vpce to deploy them, only to be able to talk to the endpoint.

Added those to make it a more real use case integration test in 44618d9